model/src/wav_history_mod.F90

```diff
@@ -372,8 +437,9 @@ subroutine write_history ( timen )
         if(vname .eq.      'PNR') call write_var2d(vname, pnr      (1:nseal_cpl) )
 
         ! Group 5
-        if (vname .eq.    'USTX') call write_var2d(vname, ust      (1:nseal_cpl)*asf(1:nseal_cpl), dir=cos(ustdir(1:nseal_cpl)), usemask='true')
-        if (vname .eq.    'USTY') call write_var2d(vname, ust      (1:nseal_cpl)*asf(1:nseal_cpl), dir=sin(ustdir(1:nseal_cpl)), usemask='true')
+        if (vname .eq.    'USTX') call write_var2d(vname, ust      (1:nsea)*asf(1:nsea), dir=cos(ustdir(1:nsea)), init0='false', global='true')
```

Comment:
@kestonsmith-noaa  - Looking at patterns here and everything else seems to be for (1:nseal_cpl) should this also be that way?

Reply:
I knew I had seen some issue w/ ust before, and it was this one https://github.com/NOAA-EMC/WW3/issues/942. So ust/ustdir/asf are dimensioned like global variables. At the time of this issue, I was testing w/ the "old" netcdf interface, which bootstrapped on top of the existing w3iogomd---meaning it was writing serial netcdf from the same root task as the binary out put was using. It does look like in this case the fields, for whatever reason, are global like group 1.